src/routes/demos/activate/index.ts

```diff
@@ -31,22 +31,48 @@ export const POST = async (requestEvent: RequestEvent) => {
     public readonly demoUuid!: string;
 
     @Expose()
-    @ValidateIf(({ obj }) => obj?.googlePrivateKey != null || obj?.googleClientCertificate != null)
+    @ValidateIf(({ obj }) => obj?.googlePrivateKey ?? obj?.googleClientCertificate)
```

Comment:
what do two question marks mean?

Reply:
It's the relatively new [The nullish coalescing (??) operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing).